### ISSUE TITLE
sun color with blackbody

### ISSUE THREAD

**Issue by mirlip** [rgsouthall/vi-suite04#2]
the function to adjust the strength and value of the blackbody material is really impressive. However, the winter values seem very orange at 10 a clock for example compared to reality. The orientation of the sky node (world material) is adjusted, but the strength stays at 1.0, maybe it's the reason why it's orange?

**Comment by rgsouthall** [comment]
I agree, although the blackbody figures are apparently realistic. I have tweaked the sun colour and  included a sky strength modifier in git. You can tweak the values yourself on line 2517 in the vi_func.py file (for sky strength) and lines 2520 & 2525 (for sun colour). If you find better values let me know.
For future reference code questions are fine here but general questions about the VI-Suite I prefer to go on the google group so I can keep things in one place.

**Comment by mirlip** [comment]
ok, thanks, will try to join the google group.
